stubs/RdKafka/KafkaConsumer.php

```diff
@@ -144,4 +154,20 @@ public function getOffsetPositions($topics)
     public function close()
     {
     }
+
+    /**
+     * @param TopicPartition[] $topic_partitions
+     * @return TopicPartition[]
+     */
+    public function pausePartitions($topic_partitions)
```

Comment:
I couldn't find it in the docs: 
https://arnaud.le-blanc.net/php-rdkafka-doc/phpdoc/class.rdkafka-kafkaconsumer.html

Is that something that was introduced recently and the official docs hasn't been updated yet?

Reply:
@kwn yep, it is in stub from base repo - https://github.com/arnaud-lb/php-rdkafka/blob/6.x/kafka_consumer.stub.php